### ISSUE TITLE
stop individual service instance

### ISSUE THREAD

**Issue by matti** [kontena/kontena#2835]
something like `kontena service stop stack/service-2` to stop instance 2.

for testing clustering (postgres stolon for example) without full node shutdown.

**Comment by SpComb** [comment]
With Kontena 1.3, the workaround would be to `docker stop ...` on the node.

With the new `ServicePodWorker` restart policies in 1.4, that won't work, the agent will just restart it. Implementing per-instance running/stopped states should be straightforward on the server, the agent doesn't need to care. Deploying the service will start all of the instances again.

**Comment by matti** [comment]
Humm? on 1.3 docker stop only works for a moment

**Comment by SpComb** [comment]
> Humm? on 1.3 docker stop only works for a moment

Right, the agent will also re-apply it on the next 30s interval. With 1.4, it just happens faster.

**Comment by jnummelin** [comment]
Needs a new endpoint `PUT /v1/services/:grid_name/:service_name/instances/:id`, or maybe `POST /v1/services/:grid_name/:service_name/instances/:id/stop`. The latter would be more inline with the apis used for `kontena service stop/start/restart`.

The CLI command could be actually incorporated with the `kontena service stop`, with an option `--instance X` as it is with `kontena service rm --instance 1 my-service`
